src/bp/core/services/dialog/dialog-engine.ts

```diff
@@ -303,9 +357,10 @@ export class DialogEngine {
       this._logTransition(event.botId, event.target, context.currentFlow, context.currentNode, transitionTo)
 
       event.state.__stacktrace.push({ flow: context.currentFlow!, node: transitionTo })
-      // When we're in a skill, we must remember the location of the main node for when we will exit
-      const isInSkill = context.currentFlow && context.currentFlow.startsWith('skills/')
-      if (isInSkill) {
+      // When we're in a sub flow, we must remember the location of the main node for when we will exit
```

Comment:
what do you mean by the main node ? calling node ?

Reply:
Hum I guess, i just changed skill for sub flow. I think parent node make more sense  in that case